product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: "Third party monitoring integrations"
+---
+
+BigAnimal provides support for third-party monitoring integrations for both Bring your own account and BigAnimal's hosted.
+
+Monitoring integrations are configured at project level in BigAnimal. It can't be turned on or off for the individual clusters. An Org admin or a Project owner can set up an integration. Only one integration can be set up per project.
+
+By default, all the integrations are disabled. Enable it using the **Integrations** tab after creating the project. 
+
+All the metrics collected from all the clusters in the Project are sent to the integrated tool and are displayed in BigAnimal's  *Monitoring and logging** tab using PEMx. The collected logs are exported to the object storage by default.
```

Comment:
"When no third party monitoring integration is enabled, ..."

and also link here to the docs on PEM-X and logs.

Reply:
@ringerc, You mean
When no third party monitoring integration is enabled, All the metrics collected from all the clusters in the Project are sent to the integrated tool and are displayed in BigAnimal's  *Monitoring and logging** tab using PEMx. The collected logs are exported to the object storage by default. ???

Reply:
No changes needed here as per the conversation on this thread: https://edb-starlight.slack.com/archives/C061LSJCJCV/p1706101353956549